backend/protocol_rpc/health.py

```
@@ -64,117 +96,334 @@ async def health_check(
+    try:
+        async with aiohttp.request(
+            "GET",
+            status_url,
+            timeout=aiohttp.ClientTimeout(total=timeout_s),
+        ) as resp:
+            if resp.status != 200:
+                return False, f"genvm_manager_status_http_{resp.status}"
+            return True, None
+    except (aiohttp.ClientError, asyncio.TimeoutError) as exc:
+        return False, f"genvm_manager_status_error: {exc}"
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)